tommath_private.h

```diff
@@ -139,7 +139,7 @@ extern void MP_FREE(void *mem, size_t size);
 #define MP_IS_EVEN(a) (((a)->used == 0) || (((a)->dp[0] & 1u) == 0u))
 #define MP_IS_ODD(a)  (((a)->used > 0) && (((a)->dp[0] & 1u) == 1u))
 
-#define MP_SIZEOF_BITS(type)    (CHAR_BIT * sizeof(type))
+#define MP_SIZEOF_BITS(type)    ((size_t)CHAR_BIT * sizeof(type))
```

Comment:
Are these casts necessary? I would prefer if you only add casts where they are strictly necessary. Otherwise they just obfuscate the code.

Reply:
@minad fine like that?

Reply:
Not a fan of the cast and I don't think it is necessary, but not really worth discussing...